content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -1,74 +1,191 @@
 
-2. Generate a validator key pair:
+For this reason, aside from getting your validator up and running, another key aspect of operating a validator is putting infrastructure in place that engenders trust in your validator. Before validation list publishers can add your validator to their UNLs, they must first have trust in your validator. Once your validator has been added to a UNL, it is a _trusted_ validator that participates in the consensus process. ***TODO: what do we mean by "validator list publishers"? Is this specifically about getting a validator listed in this blob? https://vl.ripple.com/ Why is it in a blob and can we provide guidance about how to decode it? Whatever is in this list would be reflected on XRPCharts' list of UNL validators, correct?***
+
+To run a `rippled` server as a validator, complete the following tasks:
+
+1. Install a `rippled` server. For more information, see [Install rippled](install-rippled.html).
+
+2. [Enable validation](#enable-validation-on-your-rippled-server) on the `rippled` server.
+
+Beyond this, you can optionally complete the following tasks that help to create trust in your validator:
+
+1. [Understand the traits of an ideal validator](#understand-the-traits-of-an-ideal-validator).
+
+2. [Set up a proxy to help protect your validator](#set-up-a-proxy-to-help-protect-your-validator) from [DDoS](https://en.wikipedia.org/wiki/Denial-of-service_attack) attacks.
+
+3. [Request domain verification](#request-domain-verification).
+
+
+
+## Enable Validation on Your `rippled` Server
+
+Enabling validation on your `rippled` server means providing a validator token in your server's `rippled.cfg` file. Ripple recommends using the `validator-keys` tool (included in `rippled` yum and Alien RPMs) to securely generate and manage your validator keys and tokens. ***TODO: correct reference to yum and Alien RPMs? These are the only RPMs, correct? Should we also point to https://github.com/ripple/validator-keys-tool for folks who want to manually compile and use it for a validator running on Windows?***
+
+For more information about using the `validator-keys` tool, see the [Validator Keys Tool Guide](https://github.com/ripple/validator-keys-tool/blob/master/doc/validator-keys-tool-guide.md).
+
+1. Generate a validator key pair.
 
         $ /opt/ripple/bin/validator-keys create_keys
 
-    **Warning:** Store the generated `validator-keys.json` key file in a secure but recoverable location, such as an encrypted USB flash drive. Do not modify its contents.
+    **Warning:** Store the generated `validator-keys.json` key file in a secure but recoverable location, such as an encrypted USB flash drive. Do not modify its contents. ***TODO: Is the main point here that this is an offline location? Specifically, is it not secure enough to keep it in the default location it is generated in? Like if I'm running my server in AWS EC2 - should I move the json file to my flash drive and delete it from EC2?***
```

Comment:
Yes, you want a secure offline location. If someone finds your private validator key, they can impersonate your validator and sign validations on your behalf.